sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/details/key_serializers.hpp

```diff
@@ -40,11 +40,26 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys { nam
         KeyVaultKey& key,
         Azure::Core::Json::_internal::json const& json);
 
+    std::string static inline GetUrlAuthorityWithScheme(Azure::Core::Url url)
```

Comment:
static before return type.
inline is not needed - functions defined within a class or structs are automatically inline.